stylesheets/manifest.css

```diff
@@ -1967,9 +1965,6 @@ li.entry .error-icon-container {
     .android-dark .search::selection {
       background: white;
       color: #454545; }
-    .android-dark .search::-webkit-search-cancel-button {
-      background: url("/images/x_white.svg") no-repeat center;
-      background-size: cover; }
```

Comment:
I think this won't work really well as you changed the compiled css file.
Maybe you have to try fix this in the source sass files and than compile it via running grunt in the directory.
Sass compiles to this file and as far as I know, this change would be overwritten with the next commit.